service/templates/terms_and_conditions.html

```diff
@@ -0,0 +1,167 @@
+{% extends "layout.html" %}
+{% block body %}
+
+  <a class="back-link" href="#" onclick="history.go(-1); return false;">Back</a>
```

Comment:
I think if the user has JavaScript disabled, they can just use the back button - their journey doesnt have to be optimal. What I will do though is hide this link unless you've got JS so that at least it isn't broken in a misleading way...